glue_astronomy/translators/regions.py

```diff
@@ -93,7 +105,10 @@ def roi_subset_state_to_region(subset_state, to_sky=False):
         raise NotImplementedError(f"ROIs of type {roi.__class__.__name__} are not yet supported")
 
     if to_sky:
-        reg = reg.to_sky(subset_state.xatt.parent.coords)
+        if override_wcs is not None:
+            reg = reg.to_sky(override_wcs)
+        else:
+            reg = reg.to_sky(subset_state.xatt.parent.coords)
```

Comment:
Do this at module-level imports:

```python
from astropy.wcs.wcsapi import BaseHighLevelWCS
```

Then this:

```suggestion
    if to_sky is True:
        reg = reg.to_sky(subset_state.xatt.parent.coords)
    elif isinstance(to_sky, BaseHighLevelWCS):
        reg = reg.to_sky(to_sky)
```

Reply:
done